lib/tasks/tags.rake

```diff
@@ -14,4 +14,4 @@ namespace :tags do
         end
```

Comment:
(riga 13) `Tag.find_or_create_by! tag: tag` -> `Tag.find_or_create_by!(tag: tag)`
Metterei le parentesi per consistenza con le altre chiamate a funzione.

Reply:
In realtà forse sarebbe da fixare con un'altra PR, perché non c'entra tanto con l'about us. Vedi tu.

Reply:
fixo qua